docs/cdk-erigon/deploy-testnet.md

```diff
@@ -1,208 +1,1007 @@
+http.port: 8545
+http.api: ["eth", "net", "web3", "zkevm", "txpool"] # IMPORTANT: Include "zkevm"
+private.api.addr: "localhost:9091"                 # For cdk-erigon internal/debug APIs
+
+# L2 Chain Parameters
+zkevm.l2-chain-id: {YOUR_L2_CHAIN_ID} # e.g., 2001
+
+# L1 Configuration (Sepolia)
+zkevm.l1-chain-id: 11155111 # Sepolia Chain ID
+zkevm.l1-rpc-url: "{YOUR_SEPOLIA_RPC_URL}"
+# zkevm.l1-first-block: {L1_BLOCK_NUMBER_OF_FIRST_CONTRACT_DEPLOYMENT} # Optional: for faster initial sync
+
+# L1 Contract Addresses (FROM PHASE 1)
+zkevm.address-sequencer: "{DEPLOYED_POLYGONROLLUPMANAGER_CONTRACT_ADDRESS_ON_SEPOLIA}" # This contract manages sequencer rights and receives batches.
+zkevm.address-zkevm: "{DEPLOYED_POLYGONZKEVM_CONTRACT_ADDRESS_ON_SEPOLIA}"
+zkevm.address-rollup: "{DEPLOYED_POLYGONROLLUPMANAGER_CONTRACT_ADDRESS_ON_SEPOLIA}"
+zkevm.address-ger-manager: "{DEPLOYED_POLYGONZKEVMGLOBALEXITROOT_CONTRACT_ADDRESS_ON_SEPOLIA}"
+# zkevm.address-bridge: "{DEPLOYED_BRIDGE_CONTRACT_ADDRESS_ON_SEPOLIA}" # If cdk-erigon needs it directly
+# zkevm.address-dac: "{DEPLOYED_DATACOMMITTEE_CONTRACT_ADDRESS_ON_SEPOLIA}" # If cdk-erigon interacts with L1 DAC contract
+
+# Data Stream (if cdk-node or other components consume it)
+# zkevm.data-stream-port: 9092
+# zkevm.data-stream-host: "0.0.0.0"
+
+# Sequencer Specific Config (If cdk-erigon is the sequencer)
+# zkevm.executor-urls: "http://localhost:50071" # URL of your zkevm-prover/executor mock or real
+# zkevm.executor-strict: true # Set to false for mock proving without verification (dev only)
+# zkevm.l2-sequencer-rpc-url: "http://localhost:8545" # Itself, if sequencer
+# zkevm.l2-datastreamer-url: "ws://localhost:9092" # If providing a data stream
+
+# Other settings from cdk-erigon examples (timeouts, cache, SMT, etc.)
+# ...
 ```
-
-This command creates an enclave named "cdk" and pulls the configuration directly from the GitHub repository.
-
-### Start from Local Repository (Optional)
-
-If you've cloned the kurtosis-cdk repository (works on both macOS and Linux):
-
-```bash
-# Navigate to your kurtosis-cdk directory
-cd path/to/kurtosis-cdk
-
-# Start the network from local files
-kurtosis run --enclave cdk .
+*   **Paths to Genesis**: `cdk-erigon` will look for `dynamic-{yournetwork}-allocs.json`, etc., in a directory specified by `--zkevm.genesis-config-path` or relative to its execution/config if structured correctly. The `kurtosis-cdk` Starlark scripts (`cdk_erigon.star`) show how these files are made available to the `cdk-erigon` service.
+
+### 3. `cdk-node` Configuration (Sequence Sender / Aggregator)
+
+The `cdk-node` component (from `0xPolygon/cdk` or `0xPolygonHermez/zkevm-node` if using older stack) is responsible for preparing batches of L2 transactions and submitting them to the L1. If `cdk-erigon` is not set as the sequencer, `cdk-node` can also take on the sequencer role. Additionally, if a prover is integrated, `cdk-node` can manage the aggregation of proofs.
```

Comment:
"preparing batches from the L2 into sequences to be posted to the L1" would be a better description here.